### PR TITLE
fix: establish reactivity on models and maps component for all operations

### PR DESCRIPTION
**Pull request by ChristianLieven**
This should get rid of the bug where adding or editing a model did not immediately update the table of maps and model components.

### CODE REVIEW

File: src/components/Maps.vue [hunk 1]
```diff
@@ -305,17 +308,20 @@ export default Vue.extend({
     },
     passModel(model) {
       this.selectedModel = model;
+    },
+    passMap(model) {
+      console.log(model);
```
Comment (kvikshaug) [inline src/components/Maps.vue]: This argument should probably be `map`. The `console.log` is to be replaced later?

File: src/components/Maps.vue [hunk 2]
```diff
@@ -13,7 +13,10 @@
       v-model="isModelCreationDialogVisible"
       @returnObject="passModel"
     />
-    <NewMap v-model="isMapCreationDialogVisible" />
+    <NewMap 
+      v-model="isMapCreationDialogVisible" 
+      @returnObject="passMap"
```
Comment (kvikshaug) [inline src/components/Maps.vue]: Event names are recommended [to be kebab-case](https://vuejs.org/v2/guide/components-custom-events.html#Event-Names).

File: src/components/Maps.vue (hunk 1 above)
Comment (ChristianLieven) [reply src/components/Maps.vue]: Nah this was a left over statement which I've since removed.

File: src/components/Maps.vue (hunk 2 above)
Comment (ChristianLieven) [reply src/components/Maps.vue]: True! Will fix it soon!

**Pull request merged**